src/components/CommentInfo/CommentInfo.tsx

```diff
@@ -1,7 +1,20 @@
 import React from 'react';
 
 import './CommentInfo.scss';
+import { Comment } from '../../types/interfaces';
 
-export const CommentInfo: React.FC = () => (
-  <>Put the comment here</>
-);
+type Props = {
+  comment: Comment
+};
+
+export const CommentInfo: React.FC<Props> = ({ comment }) => {
+  return (
+    <>
+      <h3 data-cy="comment-name">{ comment.name }</h3>
+      <p data-cy="comment-body">{ comment.body }</p>
+      <a href="email" data-cy="comment-email">{ comment.email }</a>
+    </>
+  );
+};
```

Comment:
```suggestion
export const CommentInfo: React.FC<Props> = ({ comment }) => (
    <>
      <h3 data-cy="comment-name">{ comment.name }</h3>
      <p data-cy="comment-body">{ comment.body }</p>
      <a href="email" data-cy="comment-email">{ comment.email }</a>
    </>
  );
```
use shorthand in such cases